### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Fleck85**
- Fixed Bug in the flightplanner (variable "loc" has no "position" value, because it already is a position)

- Fixed Bug:
AI aircraft were spawned in the air regardless of the settings

### CODE REVIEW

File: gen/aircraft.py [hunk 1]
```diff
@@ -389,7 +389,7 @@ def setup_group_activation_trigger(self, flight, group):
 
     def generate_planned_flight(self, cp, country, flight:Flight):
         try:
-            if flight.client_count == 0 and self.game.perf_ai_parking_start:
```
Comment (Fleck85) [inline gen/aircraft.py:392]: The perf_ai_parking_start variable is placed beneath "game.settings" and not directly under "game."

File: gen/flights/ai_flight_planner.py [hunk 2]
```diff
@@ -179,10 +179,10 @@ def commision_cap(self):
                 orbit1p = loc.position.point_from_heading(hdg + 90, radius)
             else:
                 loc = self.from_cp.position.point_from_heading(random.randint(0, 360), random.randint(nm_to_meter(5), nm_to_meter(40)))
-                hdg = self.from_cp.position.heading_between_point(loc.position)
```
Comment (Fleck85) [inline gen/flights/ai_flight_planner.py:182]: "loc" in this else tree is already a "Point" containing only an X and Y coordinate and does not contain the value "position".

**Comment by Khopa** [comment]
Hello Fleck85,

Thank you your contribution ! 

But in fact, I have stopped using the qt_ui branch. I was planning to delete it actually, as i've merged it on "master" on this repo and i'm now using the "develop" branch for development. And... i've just checked and these issues should already be fixed on the develop branch. 

Sorry for not communicating on this.. 😕 

But if you have familiarized yourself with the code base a bit, i'll be very happy to review and accept your contributions in the future !

Khopa

**Comment by Fleck85** [comment]
Roger that.
I may not yet be able to write stuff, but atleast i will try to help the debugging...

**Pull request closed without merging**